app/models/organization.rb

```diff
@@ -57,7 +57,7 @@ class Organization < ApplicationRecord
   validates :name, :division, :country, presence: true
 
   validate :primary_contact_is_member
-  with_options if: ->(org) { org.country.iso_code == "US" } do |us_org|
+  with_options if: ->(org) { org&.country&.iso_code == "US" } do |us_org|
```

Comment:
~~I don't believe the first `&` is needed, is it?~~

~~In general I am leery of two &.'s in a row due to Law of Demeter. In fact, two .'s is typically frowned upon, isn't it? Here it seems ok since `org` really just means `self` since it's a callback thing and really `country` is an attribute of Organization. So LoD doesn't really feel violated here.~~

~~But still, the fewer &.'s, the better in my book. And `org` should never be nil, right?~~

Reply:
Can we actually just add a non-null constraint to country_id? I see this kind of thing a lot in Madeline where safe navigation is being used on columns that shouldn't be null but are missing a non-null constraint just due to them not being included in the data model when it was first built out. I'd like us to start adding these bit by bit because they reduce errors like this and make code simpler. What do you think @meltheadorable and also @DevneyHamilton?

If we add the constraint, the &.s shouldn't be needed anymore.

Reply:
I checked the prod database and there was only 4 orgs with missing country IDs so I did a bit of sleuthing and set them manually. (3 mexico, 1 US). So the constraint shouldn't fail, at least on prod.